roles/config-rh-sso/tasks/configure-id-provider.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: "Retrieve template for Identity Provider"
+  set_fact:
+    id_provider_json: "{{ lookup('template', 'id-providers.json.j2') | to_json }}"
+
+- name: "Requesting Access Token"
+  uri:
+    url: "https://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/realms/master/protocol/openid-connect/token"
+    method: POST
+    body_format: form-urlencoded
+    body:
+      username: "{{ sso_admin_user }}"
+      password: "{{ sso_admin_pass }}"
+      grant_type: "password"
+      client_id: "admin-cli"
+    validate_certs: no
+  register: rh_sso_token
+
+- name: "Create Identity Provider for Red Hat Single Sign-On"
+  uri:
+    url: "https://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/admin/realms/master/identity-provider/instances"
```

Comment:
Just like we did for the realm role, you'll want to be able to specify a realm for each identity provider and then substitute the variable in your url

Reply:
Makes sense. Fixed it.